### ISSUE TITLE
Miscellaneous about Responsive Design and Accessibility

### ISSUE THREAD

**Issue by trob** [Fabrik/fabrik#1508]
Just a unordered collection

Radius search:
popup is displayed at site top, not visible window top (so with a large header or module above the list the popup vanishes, you have to scroll up to find it)

Filter 
is still via HTML table (-> not responsive)
labels are missing a "for" (Accessibility: "ERROR: form label missing")

Navigation:
no. of rows selection: also "ERROR: form label missing"

**Comment by pollen8** [comment]
> popup is displayed at site top, not visible window top
For me the pop up appears in the correct place now

Filter is now bootstrap dived and the labels have been fixed.

I've not done the navigation label thing though

**Comment by pollen8** [comment]
the navigation label has also been added
